public/game-master/styles/game-master.styl

```diff
@@ -0,0 +1,63 @@
+.game-master-screen
+	.user-panel
+		width: 100%
+
+.game-master-screen-container
+	width: 96%
```

Comment:
Hmmm, it looks alright in my Sublime...
Looks like there's a bug in github diff :octocat: